Application/SolutionScripts/EdFi.Ods.WebApi/EdFi.Ods.WebApi.PreRelease.nuspec

```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
Removed this folder because it was mistakenly copied and pasted from the one that's one level above.